packages/screens/DAppStore/apps/toripunks/components/buttonLabel/BuyToripunksButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Linking } from "react-native";
+
+import { ButtonLabel } from "./ButtonLabel";
+
+export const BuyToripunksButton: React.FC<{ size: "S" | "Mobile" }> = ({
+  size,
+}) => (
+  <ButtonLabel
+    text="BUY TORIPUNKS"
+    size={size}
+    actionable
+    onPress={() => {
+      Linking.openURL(
+        "https://app.teritori.com/collection/tori-tori1plr28ztj64a47a32lw7tdae8vluzm2lm7nqk364r4ws50rgwyzgsapzezt"
```

Comment:
Maybe we should put this hardcode value to a config/settings file or at least a constant

Reply:
this should at least be a relative link

Reply:
fixed as relative